gulpfile.js

```diff
@@ -255,7 +304,13 @@ gulp.task('run-test-e2e', function() {
 });
 
 gulp.task('test', function() {
-  runSequence('build', 'webserver', 'test-integration', 'stop-webserver');
+  runSequence(
+    'build',
+    'tests-unit',
+    'webserver',
+    'test-integration',
+    'stop-webserver'
+  );
```

Comment:
Comment: this `gulpfile.js` starts to get really big and messy. That's fine for now but we're going to need some refactoring at some point.